usr/sbin/pkcsslotd/slotmgr.c

```diff
@@ -791,6 +828,9 @@ int main(int argc, char *argv[], char *envp[])
             return EACCES;
     }
 
+    if (drop_capabilities() != 0)
+        return 7;
```

Comment:
what means 7? Is there probably an error constant?

Reply:
Well, the exit codes of pkcsslotd are a mess.... other places also return arbitrary numbers to somehow indicate the place where it happend.... I will leave it as is for now, but that's something I need to clean up generally at a later time.